File: Dart/q2.dart
int fibonacci_num() {
  int start = 0;
  int sec = 1;
  int result = 0;
  while (sec < 4000000) {
    int temp = start;
    start = sec;
    sec = temp + sec;
    if (start % 2 == 0) {
      result += start;
    }
  }
  return result;
}

void main() {
  print("Fibonacci Numbers");
  var result = fibonacci_num();
  print(result);
}

File: Python/q2.py
def fibonaaci(n):
    start = 0
    sec = 1
    sum = 0
    while sec < n:
        temp = start 
        start = sec
        sec = temp + start
        if(sec % 2 == 0):
            sum += sec
    return sum
size = int(input("Enter the size of the fibonacci: "))

result = fibonaaci(n=size)
print(result)